static/css/kiali_style.css

```diff
@@ -64,17 +64,69 @@ p {
 }
 
 .features-bottom-box {
-  display: block;
-  padding: 40px;
+  display: inline-block;
+  padding: 5px;
+  top: -50px;
+  position: relative;
+  width: 100%;
   background: #fff;
   box-shadow: 0 10px 40px 0 rgba(47, 47, 47, .1);
   border-radius: 3px;
   cursor: pointer
 }
 
+@media screen and (max-width: 768px) {
+  #features-list  {
+    display: inline-block;
+  }
+}
+
```

Comment:
You need to move this down the #features-list, this is compilers from top to down so L84-L89 overwrite the display.

Reply:
anyway I think that would be better set inline-grid or grid  instead inline-block

Reply:
I tried all the variations, they all look the same to me. I pushed the code.